### ISSUE TITLE
Error initializing server on Windows 11 with Node 18.4.0

### ISSUE THREAD

**Issue by UpperCod** [enhance-dev/arc-plugin-enhance#6]
Hi, I have used the installation CLI to try to use Enhance, once the installation is finished I have proceeded to start the command
`npm run start`, but the following is generated when visiting the server.

```sh

C:\git\atomico\examples\check3>npm run start

> start
> sandbox

         App ~ myproj
      Region ~ us-west-2
     Profile ~ @aws profile / AWS_PROFILE not configured
     Version ~ Sandbox 5.3.3
         cwd ~ C:\git\atomico\examples\check3

√ Sandbox @tables created in local database
√ Sandbox @http (HTTP API mode / Lambda proxy v2.0 format / live reload) routes
    any /* ................................ node_modules\@enhance\arc-plugin-enhance\src\http\any-catchall

    http://localhost:3333

√ Sandbox Started in 84ms
√ Local environment ready!

○ Sandbox Running 1 Sandbox startup plugin
√ Sandbox Ran Sandbox startup plugin in 7ms
√ Sandbox File watcher now looking for project changes

TypeError: Missing parameter name at 1
    at lexer (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:48:23)
    at parse (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:98:18)
    at stringToRegexp (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:329:27)
    at pathToRegexp (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:407:12)
    at file:///C:/git/atomico/examples/check3/node_modules/@enhance/arc-plugin-enhance/src/http/any-catchall/_get-module.mjs:24:49
    at Array.map (<anonymous>)
    at getModule (file:///C:/git/atomico/examples/check3/node_modules/@enhance/arc-plugin-enhance/src/http/any-catchall/_get-module.mjs:24:40)
    at Object.api (file:///C:/git/atomico/examples/check3/node_modules/@enhance/arc-plugin-enhance/src/http/any-catchall/router.mjs:18:18)
    at combined (C:\git\atomico\examples\check3\node_modules\@architect\functions\src\http\async\index.js:36:26)

TypeError: Missing parameter name at 1
    at lexer (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:48:23)
    at parse (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:98:18)
    at stringToRegexp (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:329:27)
    at pathToRegexp (C:\git\atomico\examples\check3\node_modules\path-to-regexp\dist\index.js:407:12)
    at file:///C:/git/atomico/examples/check3/node_modules/@enhance/arc-plugin-enhance/src/http/any-catchall/_get-module.mjs:24:49
    at Array.map (<anonymous>)
    at getModule (file:///C:/git/atomico/examples/check3/node_modules/@enhance/arc-plugin-enhance/src/http/any-catchall/_get-module.mjs:24:40)
    at Object.api (file:///C:/git/atomico/examples/check3/node_modules/@enhance/arc-plugin-enhance/src/http/any-catchall/router.mjs:18:18)
    at combined (C:\git\atomico\examples\check3\node_modules\@architect\functions\src\http\async\index.js:36:26)
```

## Additional Information

- Window 11
- Node v18.4.0
- I have not generated changes in the structure created by the CLI

I'll be there for whatever you need

**Comment by ryanblock** [comment]
Ok, bug is fixed in 1.0.32, thank you for your patience!

**Comment by UpperCod** [comment]
Works 🎉
